nailgun/entities.py

```diff
@@ -2920,6 +2952,15 @@ class Meta(object):
         api_names = {'from_': 'from'}
         server_modes = ('sat')
 
+    def create_payload(self):
```

Comment:
Can you update `tests.test_entities.CreatePayloadTestCase.test_no_attributes`, please?